locales/en/index.yml

```diff
@@ -120,29 +120,38 @@ profile:
     privacy:
       title: Privacy
       description: Export data, delete account
-      header: Informativa sul trattamento dei dati personali dei Beta Tester dell'app IO
-      updated: It is required to accept the new version of Terms and Conditions
-      text: !include profile/privacy.md
-      contextualHelpTitlePolicy: What is the information on the processing of personal data
-      contextualHelpContentPolicy: !include profile/profile_privacy_policy.md
-      contextualHelpTitle: Your privacy settings
-      contextualHelpContent: !include profile/profile_privacy_main.md
-    termsAndConditions:
-      title: Terms and Conditions
-      description: Review Terms & Conditions of the app
-    mainPrivacy:
-      screenTitle: Privacy
-      screenSubtitle: Here you will find information related to your personal data.
-      privacyPolicy:
-        title: Privacy policy
-        description: Review accepted privacy terms
+      subtitle: Here you find information related to your personal data.  
+      loading: Wait few seconds, we're checking your profile..
       removeAccount:
         title: Delete your account
         description: Remove all data associated to your account from your phone and
-          our systems (SPID required)
+          our systems
+        alert: 
+          requestTitle: Are you sure you want to delete your account?
+          requestSubtitle: All your message and persona settings will be lost.
+          oldRequest: We are already proceeding to delete your account
+          newRequest: We submitted your request to delete your account
+          confirmSubtitle: After processing, you will receive an email confirming the deletion.
+        error: An error occurred while sumbit your deletion request, please retry! 
       exportData:
         title: Export your data
         description: Send to your email all your account’s data in an open format
+        alert: 
+          requestTitle: Are you sure you want to export all your data?
+          oldRequest: We are already proceeding to export all your data
+          newRequest: We submitted your request to export all your data
+          confirmSubtitle: After processing, you will receive an email containing all your data.
+        error: An error occurred while submit your exporting request, please retry!
+      privacyPolicy:
+        title: Privacy policy
+        description: Review the accepted privacy terms
+        header: Informativa sul trattamento dei dati personali dei Beta Tester dell'app IO
```

Comment:
this is italian

Reply:
updated within https://github.com/pagopa/io-app/pull/1584/commits/54c95d7824b87810bb4f4fc5d2e6edb167e45554